sql/updates/world/master/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `creature_template`
+	RENAME COLUMN `rank` TO `Classification`;
```

Comment:
This doesnt work with mysql 5.7 (theoretically not supported but we didnt drop it from TC yet), you need to use `CHANGE` command instead of `RENAME`

Reply:
Is it being dropped anytime soon? 👀

Reply:
Not sooner than this PR gets reviewed/merged

Reply:
I'll use change then 😢